teoriasegnali/variabili-aleatorie.tex

```diff
@@ -19,7 +19,7 @@ \section{Funzione distribuzione di probabilità}
 \item $0\leq F_X(x)\leq 1$
 \item $\lim\limits_{x\to+\infty}F_X(x)=1$
 \item $\lim\limits_{x\to-\infty}F_X(x)=0$
-\item $F_X$ è monotona non decrescente: \[x_1<x_2\implies F_X(x_1)\leq F_X(x_2)\] 
+\item $F_X$ è monotona non decrescente: \[x_1<x_2\implies F_X(x_1)\leq F_X(x_2)\]
```

Comment:
I can't see diff
may be a CRLF / LF mismatch caused by repo setting?
I will try to fix it with .gitattributes